services/ordering/src/EShopOnAbp.OrderingService.Application/Orders/OrderAppService.cs

```diff
@@ -106,8 +106,7 @@ private OrderDto CreateOrderDtoMapping(Order order)
             Id = order.Id,
             OrderNo = order.OrderNo,
             OrderDate = order.OrderDate,
-            OrderStatus = order.OrderStatus.Name,
-            OrderStatusId = order.OrderStatus.Id,
+            OrderStatus = order.OrderStatus.ToString(),
```

Comment:
.ToString() won't be required when we change type in dto
```suggestion
            OrderStatus = order.OrderStatus,
```